packages/framework-provider-azure-infrastructure/src/infrastructure/synth/TerraformContainers.ts

```diff
@@ -0,0 +1,53 @@
+import { TerraformStack } from 'cdktf'
+import { CosmosdbAccount, CosmosdbSqlContainer, CosmosdbSqlDatabase, ResourceGroup } from '@cdktf/provider-azurerm'
+import { toTerraformName } from '../helper/utils'
+import { BoosterConfig } from '@boostercloud/framework-types'
+import { eventsStoreAttributes } from '@boostercloud/framework-provider-azure'
+
+export class TerraformContainers {
+  static build(
+    terraformStack: TerraformStack,
+    resourceGroup: ResourceGroup,
+    appPrefix: string,
+    cosmosdbDatabase: CosmosdbAccount,
+    cosmosdbSqlDatabase: CosmosdbSqlDatabase,
+    config: BoosterConfig
+  ): Array<CosmosdbSqlContainer> {
+    const idEvent = toTerraformName(appPrefix, 'events')
+    const cosmosdbSqlEventContainer = new CosmosdbSqlContainer(terraformStack, idEvent, {
+      name: config.resourceNames.eventsStore,
+      resourceGroupName: cosmosdbDatabase.resourceGroupName,
+      accountName: cosmosdbDatabase.name,
+      databaseName: cosmosdbSqlDatabase.name,
+      partitionKeyPath: `/${eventsStoreAttributes.partitionKey}`,
+      partitionKeyVersion: 2,
+      throughput: 400,
+    })
+
+    const readModels = Object.keys(config.readModels).map((readModel) =>
+      this.createReadModel(terraformStack, config, readModel, cosmosdbDatabase, cosmosdbSqlDatabase)
+    )
+
+    return [cosmosdbSqlEventContainer].concat(readModels)
```

Comment:
Maybe separate the creation of the event container into another function and make this `build` method only do the `concat`?

Reply:
Yeah. I'm going to do something similar, let me change it and tell me what you think about it